tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/cluster/distribution/ClusterTestBase.java

```diff
@@ -723,7 +723,7 @@ protected void sendInRange(final int node,
                               final int msgEnd,
                               final boolean durable,
                               final String filterVal) throws Exception {
-      sendInRange(node, address, msgStart, msgEnd, durable, filterVal, null);
```

Comment:
these changes don't seem to be necessary for the "redistribute to old bindings" use case?

Reply:
You are right, it is not necessary. I will fix it shortly

Reply:
it looks like the sendInRange change is still present